src/app/GitCommands/Patches/PatchProcessor.cs

```diff
@@ -39,7 +39,7 @@ private enum PatchProcessorState
         [GeneratedRegex(@$"(---|\+\+\+) [""]?[^/\s]+/(?<filename>.*)[""]?", RegexOptions.ExplicitCapture)]
         private static partial Regex FileNameRegex();
 
-        [GeneratedRegex(@$"^({_escapeSequenceRegex})?[ -+@]", RegexOptions.ExplicitCapture)]
+        [GeneratedRegex(@$"^({_escapeSequenceRegex})?[-+@ ]", RegexOptions.ExplicitCapture)]
```

Comment:
To further clarify

```suggestion
        [GeneratedRegex(@$"^({_escapeSequenceRegex})?[\-+@ ]", RegexOptions.ExplicitCapture)]
```

I searched for other occurrences, found one other, with no importance.
Also some testcases.
Pushed to private branch, suggest it is included